pkg/meta/version.go

```diff
@@ -6,7 +6,7 @@ const (
 	InstanceManagerAPIMinVersion = 1
 
 	// InstanceManagerProxyAPIVersion is used for compatibility check for longhorn-manager
-	InstanceManagerProxyAPIVersion    = 4
```

Comment:
This API version is already bumped for v1.5.0. The issue ticket number is 5376

Reply:
I do not anticipate this change being added to v1.5.0, so I am thinking an update is still required.

Reply:
Oh OK. Then bumping up the version is required. BTW, should we increase `CLIAPIVersion` number either?

Reply:
We handle this in longhorn-engine, unless there is somewhere else it is also necessary?

https://github.com/longhorn/longhorn-engine/blob/3af37b9e5bf984204c440fc9aa6ce9badbd0c8ad/pkg/meta/version.go#L5